sources/views/contacts/contactList.js

```diff
@@ -0,0 +1,42 @@
+import {JetView} from "webix-jet";
+import {dataContacts} from "../../models/contacts";
+
+export default class ContactList extends JetView {
+	config() {
+		return {
+			view: "list",
+			localId: "contactList",
+			select: true,
+			width: 250,
+			autoheight: false,
+			template: "<div class='container-for-image'><img class='user-avatar' src=#Photo# /> </div><div class='user-info'><p> #value# </p><p><small> #Company# </small></p></div>",
+			type: {
+				height: "auto"
+			}
+		};
+	}
+
+	init(view, url) {
+		this.listComponents = this.$$("contactList");
+		this.listComponents.attachEvent("onAfterSelect", id => this.setIdIntoUrl(id));
+		dataContacts.waitData.then(() => {
+			this.listComponents.sync(dataContacts);
+
+			const idFromUrl = url[0].params.id;
+			if (dataContacts.getItem(idFromUrl)) {
+				this.listComponents.select(idFromUrl);
+			}
+			else if (!idFromUrl && dataContacts.count()) {
+				this.listComponents.select(this.listComponents.getFirstId());
+			}
+			else {
+				this.show("./contacts");
+				webix.message("Please choose the contact");
+			}
+		});
+	}
+
+	setIdIntoUrl(id) {
+		this.show(`./contacts?id=${id}`);
```

Comment:
Parameters for Contacts should be set in Contacts (not in the nested JetView, even if it's static).
Explicit operations on a parent/nested JetViews are not recommended. 
You can trigger the needed action in a parent JetView with a custom app event, for example.
I.e.
```
List selection -> (app event)/(changes in a service)/(or in a model) -> change parameter in the parent view 
```